### ISSUE TITLE
Flash erase failure

### ISSUE THREAD

**Issue by kent-williams** [pelrun/novena-gg-tools#1]
Hello,

I mistakenly used your flasher in hopes that it would fix another issue on the senoko pvt, and now the gg is not working.

I get the following output when trying to run.

_Flash length: 2048_
_Erasing rows 0-1..._
_Verify erased row failed_
_Retrying erase..._
_Verify erased row failed_
_Retrying erase..._
_Verify erased row failed_
_Retrying erase..._
_Verify erased row failed_
_Retrying erase..._
_Verify erased row failed_
_Retrying erase..._
_Verify erased row failed_
_Retrying erase..._
_Verify erased row failed_
_Retrying erase..._

Any insight on how to fix would be much appreciated!

**Comment by kent-williams** [comment]
Somehow the gg chip came back online. I won't be retrying to flash it though now that is has.

**Comment by pelrun** [comment]
If not even the first erase command works, then you should be pretty safe (it generally either works or it doesn't do anything.) It's really only once an erase succeeds that you need to make sure that you get a full rewrite to succeed before restarting.

Also, it's tricky to power down the gg if it gets in an odd state - you have to pull the power _and_ disconnect the battery to properly reset it.

**Comment by kent-williams** [comment]
Thanks for the insight!

I think the attempt to erase it put it in a broken state, which was resolved when fully disconnecting the battery.
